game/webserver.cc

```diff
@@ -221,69 +220,69 @@ http_server::response WebServer::GETresponse(const http_server::request &request
 http_server::response WebServer::POSTresponse(const http_server::request &request) {
 	Game * gm = Game::getSingletonPtr();
 	if (request.destination == "/api/add") {
-			m_songs.setFilter("");
+		m_songs.setFilter("");
 		boost::shared_ptr<Song> pointer = GetSongFromJSON(request.body);
 		if (!pointer) {
 			return http_server::response::stock_reply(
-			http_server::response::ok, "failure");
+													  http_server::response::ok, "failure");
```

Comment:
long indent.

Reply:
Our coding style specifies that a small indent of two spaces should be used for splitting lines etc. If anything is aligned with more than 4 spaces (as was the case in the original code here), most auto-indenting tools will fail like this.

Recommendation: be careful when using auto-indent tools, and follow the coding style (don't align anything)

Reply:
it's still pretty ridiculous that I actually reviewed the entire 54-files long diff..... so I think I highlighted all errors...

Reply:
Hahahahah. Ok I'll take care of it later.